.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -70,7 +70,7 @@ chmod 550 ${SUBMITTY_INSTALL_DIR}/sbin/authentication.py
 chmod 555 ${SUBMITTY_INSTALL_DIR}/sbin/killall.py
 
 # DAEMON_USER only things
-array=( build_config_upload.py run_lichen_plagiarism.py sendEmail.py submitty_autograding_shipper.py submitty_autograding_worker.py submitty_daemon_jobs autograder)
+array=( build_config_upload.py run_lichen_plagiarism.py submitty_autograding_shipper.py submitty_autograding_worker.py submitty_daemon_jobs autograder )
```

Comment:
don't we still want to chmod this to proper permissions?

Reply:
Yep. Forgot to re-add that in after the name of the email script changed. Good catch.